x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../types';
+import { checksFactory } from '../saved_objects';
+
+/**
+ * Routes for job service
```

Comment:
Looks like a lot of copy and paste text in here that needs rewriting for these saved objects routes.